maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/HardshipService.java

```diff
@@ -37,9 +39,11 @@ public void update(WorkflowRequest request) {
         hardshipMapper.performHardshipResponseToApplicationDTO(apiPerformHardshipResponse, request.getApplicationDTO());
     }
 
-    public ApiPerformHardshipResponse rollback(WorkflowRequest request) {
-        ApiPerformHardshipRequest performHardshipRequest =
-                hardshipMapper.workflowRequestToPerformHardshipRequest(request);
-        return hardshipApiService.rollback(performHardshipRequest);
+    public void rollback(WorkflowRequest request) {
+        HardshipReviewDTO hardshipReviewDTO = hardshipMapper.getHardshipReviewDTO(request.getApplicationDTO());
+        hardshipApiService.rollback(hardshipReviewDTO.getId());
+        hardshipReviewDTO.setReviewResult(null);
```

Comment:
hardshipAPIService.rollback() set the review result to null and status to IN_ROGRESS. Do we need to set the same to the hardshipReviewDTO as APIClientException is thrown after the rollback in the calling catch block?